docs/tutorials/excel-read-tutorial.md

```diff
@@ -67,10 +67,9 @@ Over the rest of the tutorial, we will normalize this data using a script. First
     ```
 
 6. Run the script.
-7. Open the console. Go to the **Ellipses** menu and press **Logs...**.
-8. You should see `[Array[1]]` in the console. This is not a number because ranges are two-dimensional arrays of data. That two-dimensional range is being logged to the console directly. Luckily, the Code Editor does let you see the contents of the array.
-9. When a two-dimensional array is logged to the console, it groups column values under each row. Expand the array log by pressing the blue triangle.
-10. Expand the second level of the array by pressing the newly revealed blue triangle. You should now see this:
+7. You should see `[Array[1]]` in the console. This is not a number because ranges are two-dimensional arrays of data. That two-dimensional range is being logged to the console directly. Luckily, the Code Editor does let you see the contents of the array.
```

Comment:
```suggestion
7. You should see `[Array[1]]` in the console. This is not a number because ranges are two-dimensional arrays of data. That two-dimensional range is being logged to the console directly. Luckily, the Code Editor lets you see the contents of the array.
```